### DISCUSSION TITLE
deploy to heroku

### DISCUSSION THREAD

**Discussion by YuMingLiao** [agentm/project-m36#341] [General]
I have managed to run a project-m36-websocket-server on heroku.

I have tested that websocket-client.html can connect to ws://tranquil-brook-74484.herokuapp.com:80/ 
database: data. (it's just a proof of concept, I might to go run something else soon.)

(Heroku doesn't have free persistent storage by default, but it is enough for the demo.)

And Here is the (maybe oversimplified) [journey](https://project-m36-fun.blogspot.com/2022/04/a-journey-to-project-m36-on-heroku.html) for someone who may need it.

**Comment by agentm** [comment]
Very cool! Why is necessary to set "dontStrip" to false?

Would you be willing to share the complete recipe including the deployment script?

**Reply by YuMingLiao** [reply]
> Very cool! Why is necessary to set "dontStrip" to false?

I tried again... Well, It turns out that it doesn't matter. Just some legacy code of trying.

> Would you be willing to share the complete recipe including the deployment script?

Sure, [here](https://github.com/YuMingLiao/blog/tree/gh-pages/project-m36-on-heroku)
